scripted-pipeline/src/test/resources/projects/testInstall/archetype.properties

```diff
@@ -2,3 +2,4 @@ groupId=org.sample
 artifactId=testArtifact
 version=1.0-SNAPSHOT
 package=io.jenkins.plugins.sample
+hostOnJenkinsGitHub=true
```

Comment:
Does this matter here?

Reply:
the value doesn't matter but it has to be set so that the tests run from what I remember